mealsharingdotnet/Program.cs

```diff
@@ -0,0 +1,131 @@
+
+using System.Linq;
+using System.Text.Json;
+using Microsoft.AspNetCore.Mvc;
+
+var builder = WebApplication.CreateBuilder(args);
+builder.Services.AddScoped<IMealService, FileMealService>();
+builder.Services.Remove(ServiceDescriptor.Transient<IMealService, FileMealService>());//delete meal service
```

Comment:
I'm curious about the reasoning behind this line. Was something not working until you have not added this?

Reply:
i was testing. primarily, i thought that like add service there might have also deelte, update services as well. 
i checked it and  it works without this line ! i forgot to remove it. thank you so much !

Reply:
Nice to see that you are exploring stuff 👍